terraform/main.tf

```diff
@@ -34,13 +34,25 @@ module "dns" {
   primary_domain = "pypi.org"
   user_content_domain = "pythonhosted.org"
 
-  google_verification = [
+  apex_txt = [
     "google-site-verification=YdrllWIiutXFzqhEamHP4HgCoh88dTFzb2A6QFljooc",
-    "google-site-verification=ZI8zeHE6SWuJljW3f4csGetjOWo4krvjf13tdORsH4Y"
+    "google-site-verification=ZI8zeHE6SWuJljW3f4csGetjOWo4krvjf13tdORsH4Y",
+    "v=spf1 include:_spf.google.com include:amazonses.com -all"
```

Comment:
Did we not have SPF set up previously?

Reply:
only for `mail.pypi.org`

Reply:
That didn't jive with my memory of it, so I just dug in real quick. It appears that by default SES uses a ``MAIL FROM`` that is set to amazonses.com, so it doesn't require a SPF check at all (see https://docs.aws.amazon.com/ses/latest/DeveloperGuide/spf.html).

So this seems fine to me, the ``include:amazonses.com`` is unneeded, but doesn't hurt. We might want to consider if we want to configure ``MAIL FROM`` to not use ``amazonses.com``, but that's not really related to this change.